packages/zosjobs/src/cli/list/jobs/Jobs.handler.ts

```diff
@@ -32,8 +32,8 @@ export default class JobsHandler extends ZosmfBaseHandler {
     public async processCmd(params: IHandlerParameters): Promise<void> {
 
         // Obtain the list of jobs - by default uses the session user and * for owner and prefix.
-        const prefix: string = (params.arguments.owner != null) ? params.arguments.owner : this.mSession.ISession.user;
-        const owner: string = (params.arguments.prefix != null) ? params.arguments.prefix : JobsConstants.DEFAULT_PREFIX;
+        const owner: string = (params.arguments.owner != null) ? params.arguments.owner : this.mSession.ISession.user;
```

Comment:
Hey, can you take a look at `GetJobs.ts`... I think the call `GetJobs.getJobsByOwnerAndPrefix` is wrong here too? The method signature:
```
getJobsByOwnerAndPrefix(session: AbstractSession, owner: string, prefix: string)
```

Reply:
Hi @tucker01 . Thanks for spotting this. I've done the changes but pipeline is not running. Already raised the issue on slack.